dashboard/test/controllers/transfers_controller_test.rb

```diff
@@ -3,284 +3,228 @@
 class TransfersControllerTest < ActionController::TestCase
   include Devise::Test::ControllerHelpers
 
+  # As we no longer generate codes with vowels, this code should not already
+  # exist in the test DB.
+  NONEXISTENT_SECTION_CODE = 'AEIOUY'.freeze
+
   setup do
     @teacher = create(:teacher)
     sign_in(@teacher)
 
     @word_section = create(:section, user: @teacher, login_type: 'word')
-    @word_user_1 = create(:follower, section: @word_section).student_user
+    @word_student = create(:follower, section: @word_section).student_user
 
     @picture_section = create(:section, user: @teacher, login_type: 'picture')
-    @picture_user_1 = create(:follower, section: @word_section).student_user
```

Comment:
Presumably the `@picture_user_1` was meant to be enrolled in the `@picture_section`. Being enrolled in `@word_section` masked some (should-be) failures below.